<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>331</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="login_button">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="name">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="text">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>186</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>გთხოვთ შეიყვანოთ სახელი და პაროლი</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="from_label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>from currency</string>
      </property>
     </widget>
     <widget class="QComboBox" name="from_box">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ლარი</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>დოლარი</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ევრო</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="to_label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to currency</string>
      </property>
     </widget>
     <widget class="QComboBox" name="to_box">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>150</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>ლარი</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>დოლარი</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ევრო</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="convert_button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>220</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>კონვერტირება</string>
      </property>
     </widget>
     <widget class="QLabel" name="empty_label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>გასუფთავება</string>
      </property>
     </widget>
     <widget class="QPushButton" name="logout_button">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>320</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
